spec/liquid_tags/youtube_tag_spec.rb

```diff
@@ -26,6 +28,20 @@ def generate_new_liquid(id)
       expect(liquid).to include('src="https://www.youtube.com/embed/fhH5xX_yW6U?start=300"')
     end
 
+    it "accepts valid YouTube ID with starting time as integer" do
+      liquid = generate_new_liquid(valid_id_with_time_num).render
+
+      expect(liquid).to include('<iframe')
+      expect(liquid).to include('src="https://www.youtube.com/embed/fhH5xX_yW6U?start=300"')
+    end
+
+    it "accepts valid YouTube ID with starting time in seconds" do
```

Comment:
Quick, non-blocking suggestion (same as above!): since there are multiple expectations within in this block, let's aggregate the failures here like so:
```suggestion
    it "accepts valid YouTube ID with starting time in seconds", :aggregate_failures do
```